### ISSUE TITLE
Start Building Base Logic For Activities Page

### ISSUE THREAD

**Issue by deeheber** [risachi/travel-project#31]
Write out Event Listeners for each checkbox.

**Comment by risachi** [comment]
Do you have an idea as to the logic of how it will work yet?

**Comment by deeheber** [comment]
PR #34

I wasn't able to figure out how to access the objects in the setup.js file, but I was able to add checks via alert boxes to test if the input checkboxes are checked or unchecked.

**Comment by risachi** [comment]
Just call App.allActivities to access the objects - that method will spit all the objects back out at you as an array, sorted by name in alphabetical order

**Comment by deeheber** [comment]
@risachi sounds good. We can dig into that next, probably easier to just increment it one tiny step at a time.

If anyone has some time and wants to continue building out the logic from where I left off, feel free to make another issue and be sure to pull down the changes once my pull request is merged to avoid merge conflicts. 

cc: @jacobshillman @Randallonius

**Comment by risachi** [comment]
Great idea, I'll write out some issues with some ideas for taking it further
